microceph/ceph/osd.go

```diff
@@ -196,7 +196,8 @@ func checkEncryptSupport() error {
 	// Check if we have the dm_crypt module
 	inf, err := os.Stat("/sys/module/dm_crypt")
 	if err != nil || inf == nil || !inf.IsDir() {
-		return fmt.Errorf("Missing dm_crypt module: %w", err)
+		helper := fmt.Sprint("Use \"sudo snap connect microceph:dm-crypt\" to enable disk encryption.")
+		return fmt.Errorf("Missing dm_crypt module: %w\n%s", err, helper)
```

Comment:
That's not quite right -- if the dm-crypt snap connection is missing it will surface as an EPERM using the cryptsetup command, even though the actual dm_crypt kernel module will be present.

You will need to check for the missing dm-crypt snap connection separately; suggest to try probing with `snapctl is-connected dm-crypt` for this; also cf. https://forum.snapcraft.io/t/add-dm-crypt-interface-to-microceph/34995/4?u=petersabaini